NewEvaluation.css

```diff
@@ -54,8 +57,29 @@ body{
 }
 
 
+.input-candidate{
+	width:33%;
+	border: 0;
+	padding: 0;
+	vertical-align: middle;
+	height: 2em;
+	/*box-shadow: 0px 1px 0px #888888; */
+}
+.input-candidate::placeholder{
+		border-bottom: 1px solid lightgrey;
+		columns: lightgrey;
```

Comment:
Q1:wrrrreeeeeeeeeeee , my bad auto fill from c+enter on sublime. 

Q2:Then for placeholder is pretty much a way that a friend of mine saw is wrong in my project(the underlining that I used with  box-shadow: 0px 1px 0px #888888; ^the comment above). This was the way to underline the way we wanted the input boxes. And sadly this is working only on Chrome as I mentioned and not on Mozilla.